src/libs/protocols/protos/agent.proto

```diff
@@ -366,7 +366,8 @@ message OnlineCPUMemRequest {
 	// resources are connected asynchronously and the agent returns immediately.
 	bool wait = 1;
 
-	// NbCpus specifies the number of CPUs that were added and the agent has to online.
+	// NbCpus specifies the number of CPUs that should be onlined in the guest.
+	// Special value 0 means agent will skip this check.
```

Comment:
Where do you use `0` to skip the check in this PR?

Reply:
in L266-L279 src/agent/src/sandbox.rs